lldb/unittests/Expression/ValueMatcher.h

```diff
@@ -0,0 +1,153 @@
+//===-- ValueMatcher.h ----------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+/// \file
+/// This file contains the definition of the ValueMatcher class which is a used
+/// to match lldb_private::Value in gtest assert/expect macros. It also contains
+/// several helper functions to create matchers for common Value types.
```

Comment:
It would be nice to include a link here to https://google.github.io/googletest/gmock_cook_book.html#writing-new-monomorphic-matchers for when someone looks at this file and wants to implement another matcher.